CSS modules: separate out unsupported features

estelle opened the pull request:
took a look at documented features that have no support

estelle committed:
CSS modules: separate out unsupported features

estelle committed:
typo

Josh-Cena reviewed (commented):
You are missing:

- https://developer.mozilla.org/en-US/docs/Web/CSS/image-resolution
- https://developer.mozilla.org/en-US/docs/Web/CSS/line-height-step

I think we should have a uniform template for noting non-support in module landing pages and other pages, and apply it to all other existing notes

chrisdavidmills reviewed (changes requested):
@estelle a few bits for you to consider, but nothing major. Nice work.

estelle commented on files/en-us/web/css/css_selectors/index.md:
```diff
@@ -18,21 +18,21 @@ Selectors, whether used in CSS or JavaScript, enable targeting HTML elements bas
 - `,` ([Selector list](/en-US/docs/Web/CSS/Selector_list))
 
+> [!NOTE]
+> The CSS selectors module introduces the [column combinator](/en-US/docs/Web/CSS/Column_combinator) (`||`) that has not yet been implemented.
```
They should definitely be consistent. Notes are to alert people to things, and stuff that is written but not implemented is more of an aside. i am going to de-note them in the module landing pages.

estelle committed:
Merge branch 'main' into 226

estelle committed:
remove notes in module landing pages. Make 'not supported' uniform across all CSS

estelle commented on files/en-us/web/css/css_fonts/index.md:
```diff
@@ -70,8 +70,8 @@ The specification also defines the `font-width` property, which is not yet suppo
-  - : Descriptor:
-    - {{cssxref("@font-feature-values/font-display", "font-display")}}
+
+The specification also defines the {{cssxref("@font-feature-values/font-display", "font-display")}} descriptor which is not yet supported by any browser.
```
I changed all the pages to make them uniform. Basically:

> The X module also introduces Y. Currently, no browsers support this/these feature(s).

it is possible that they are implemented behind a flag, so didn't want to use that lingo (pedantic).

estelle commented on files/en-us/web/css/css_selectors/index.md:
```diff
@@ -98,6 +96,9 @@ Selectors, whether used in CSS or JavaScript, enable targeting HTML elements bas
 - [Universal selectors](/en-US/docs/Web/CSS/Universal_selectors)
 
+> [!NOTE]
+> The CSS selectors module introduces the {{CSSXref(":blank")}}, {{CSSXref(":current")}}, {{CSSXref(":local-link")}}, and {{CSSXref(":target-within")}} pseudo-classes, which have not yet been implemented.
```
agree. Going with "support" instead of "implement" for pedantic reasons.

estelle committed:
remove no support for stretch and contain

estelle committed:
final tweaks

estelle commented:
> You are missing:
> 
> * https://developer.mozilla.org/en-US/docs/Web/CSS/image-resolution
> * https://developer.mozilla.org/en-US/docs/Web/CSS/line-height-step
> 
> I think we should have a uniform template for noting non-support in module landing pages and other pages, and apply it to all other existing notes

line-height-step is in a spec that has not been implemented and that we haven't documented. I added a link to the spec in the see also to help my future self find the spec and document the module. We may want to delete that page (but not in this PR). 

Fixed image-resolution. thanks for finding that one.

estelle committed:
final final tweaks

estelle committed:
final final final tweaks

estelle commented:
> @estelle a few bits for you to consider, but nothing major. Nice work.

I made everything uniform, so touched almost all the content. It may be easier to re-review rather than check previous comments individually.

chrisdavidmills commented on files/en-us/web/css/css_generated_content/index.md:
```diff
@@ -110,8 +110,7 @@ Click "Play" in the example above to see or edit the code in the MDN Playground.
 
-> [!NOTE]
-> The CSS generated content module introduces four at-risk properties that have not been implemented: `string-set`, `bookmark-label`, `bookmark-level`, and `bookmark-state`.
+The CSS generated content module also introduces four at-risk properties, including `string-set`, `bookmark-label`, `bookmark-level`, and `bookmark-state`. Currently, no browsers support these features.
```
```suggestion
The CSS generated content module also introduces four at-risk properties: `string-set`, `bookmark-label`, `bookmark-level`, and `bookmark-state`. Currently, no browsers support these features.
```

chrisdavidmills commented on files/en-us/web/css/css_paged_media/index.md:
```diff
@@ -22,8 +22,7 @@ The **CSS paged media** module defines the properties that control the presentat
 
-> [!NOTE]
-> CSS paged media module introduces two `@page` descriptors that have not been implemented: `bleeds` and `marks`.
+CSS paged media module also introduces the `bleeds` and `marks` `@page` descriptors. Currently, no browsers support these features.
```
```suggestion
The CSS paged media module also introduces the `bleeds` and `marks` `@page` descriptors. Currently, no browsers support these features.
```

chrisdavidmills reviewed (approved):
@estelle Very nice! Just a couple more suggestions for consistency, but approving.

chrisdavidmills commented:
> > @estelle a few bits for you to consider, but nothing major. Nice work.
> 
> I made everything uniform, so touched almost all the content. It may be easier to re-review rather than check previous comments individually.

Yeah, that's what I did, so all good.

estelle committed:
Update files/en-us/web/css/css_paged_media/index.md

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

estelle committed:
Update files/en-us/web/css/css_generated_content/index.md

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>
